RenderWareBuilders/RenderWareBuilder.cs

```diff
@@ -5,16 +5,19 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Numerics;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("RenderWareIo.Mta")]
```

Comment:
Not a fan of using this to expose internals. I would rather see public `IReadOnlyCollection`s for the private collection.